src/util/util_errors.h

```diff
@@ -21,9 +21,13 @@
 #ifndef __SSSD_UTIL_ERRORS_H__
 #define __SSSD_UTIL_ERRORS_H__
 
+#include "config.h"
+
 #ifndef HAVE_ERRNO_T
 #define HAVE_ERRNO_T
 typedef int errno_t;
+#else
+#include <errno.h>
```

Comment:
Why? 'errno.h' doesn't provider `errno_t` (it provides `error_t`)

Reply:
Ok, likely because of
```
AC_CHECK_TYPES([errno_t], [], [], [[#include <errno.h>]])
```

Reply:
On FreeBSD errno_t is `typedef`ed right in errno.h

Reply:
What Python version do you have?

```
grep _POSIX_C_SOURCE python3.12/Python.h
```
 - nothing

Reply:
Nah, let's leave it as it is and we'll patch this place downstream.

Reply:
`grep _POSIX_C_SOURCE /usr/local/include/python3.11/Python.h`, also gives nothing

Reply:
Ah, found:
```
python3.12/pyconfig-64.h:1875:#define _POSIX_C_SOURCE 200809L
```